plugins/klaro-config.js

```diff
@@ -0,0 +1,156 @@
+export const klaroConfig = {
+  testing: false,
+  elementID: 'eu-klaro',
+  storageMethod: 'cookie',
+  cookieExpiresAfterDays: 15,
+  translations: {
+    en: {
+      consentNotice: {
+        description: 'Hi! Could you please enable some additional services for {purposes}? You can always change or withdraw your consent after 15 days.'
+      },
+      purposes: {
+        performance: {
+          title: 'Performance Optimization',
+          description: 'These services process personal information to optimize the service that this website offers.'
+        },
+        marketing: {
+          title: 'Analytics and Feedback',
+          description: 'These services process personal information to help us better understand user behavior by tracking user flows and engagement.'
+        },
+        service: {
+          title: 'Service Provision',
+          description: 'These services are essential for the correct functioning of this website. You cannot disable them here as the service would not work correctly otherwise.'
+        }
+      }
+    }
+  },
+  services: [
+    {
+      name: 'searchResultsView',
+      purposes: ['performance'],
+      cookies: [
+        'searchResultsView'
+      ],
+      translations: {
+        en: {
+          title: 'Search results view'
+        }
+      }
+    },
+    {
+      name: 'debugSettings',
+      purposes: ['performance'],
+      cookies: [
+        'debugSettings'
+      ],
+      translations: {
+        en: {
+          title: 'Debug toggle'
+        }
+      }
+    },
+    // Jira service desk
+    // {
+    //   name: '',
+    //   purposes: ['performance'],
+    //   cookies: [
+    //     ''
+    //   ],
+    //   translations: {
+    //     en: {
+    //       title: 'Jira service desk'
+    //     }
+    //   }
+    // },
+    {
+      name: 'Google Analytics',
+      purposes: ['marketing'],
+      cookies: [
+        '_ga',
+        '_gid',
+        '_gat_UA-12776629-19'
```

Comment:
Replace this with something like 

`cookies: [
        /^_ga(_.*)?/,
        '_gid'
      ],`